releasenotes/notes/discrete-rotations-5a77bda4ddd5e77e.yaml

```
@@ -0,0 +1,7 @@
+---
+features_transpiler:
+  - |
+    Added a new transpiler pass :class:`.DiscretizeRotations`.
+    It takes a circuit containing single-qubit rotation gates, namely,
+    :class:`.RZGate`, :class:`.RXGate` and :class:`.RYGate`, whose angles are integer multiplications
+    of 2*pi/8, and converts them into discrete sets of Clifford, :class:`.TGate` and :class:`.TdgGate` gates.
```

A small nitpick, but this sounds as if the pass _only_ works for circuits with these types of gates. So maybe something like this:

```suggestion
    that converts single-qubit :class:`.RZGate`, :class:`.RXGate` and :class:`.RYGate` rotations gates
    with angles are integer multiples of pi/4, into discrete sets of
    Clifford, :class:`.TGate` and :class:`.TdgGate` gates.
```

Would you also like to include an example illustrating this pass?

---

done in [b808708](https://github.com/Qiskit/qiskit/pull/15321/commits/b80870884d27cee66f5150237e75f7ee5968fb78)